src/nginx.py

```diff
@@ -53,17 +55,21 @@ def defaults(self) -> dict:
         If a configuration is present in both, the custom configuration overrides
         the default.
 
-        Returns:
+        Returns
+        -------
             dict: The merged defaults dictionary.
+
         """
         return {k: self.custom.get(k, v) for k, v in self.charm_default.items()}
 
     @property
     def extra(self) -> dict:
         """Returns the configurations that are exclusively in the custom configuration.
 
-        Returns:
+        Returns
+        -------
```

Comment:
Is this what the cool kids use these days?

Reply:
must be from `ruff`?  I don't remember adding these myself